react-front-end/tsrc/util/BrowserStorageUtil.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to The Apereo Foundation under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * The Apereo Foundation licenses this file to you under the Apache License,
+ * Version 2.0, (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const localStorage = window.localStorage;
+
+/**
+ * Retrieve data from browser local storage by a key. Return undefined if
+ * no data is found.
+ *
+ * @param key Key of the data to be retrieved.
+ */
+export const getDataFromLocalStorage = (key: string): string | null =>
+  localStorage.getItem(key);
+
+/**
+ * Save data to browser local storage.
+ * @param key Key of the data.
+ * @param value Value of the data.
+ */
+export const setDataToLocalStorage = (key: string, value: string) =>
```

Comment:
Feels more idiomatic to go with:

```suggestion
export const saveToLocalStorage = (key: string, value: string) =>
```